src/assets/stylesheets/ui-root.scss

```diff
@@ -223,6 +223,253 @@ body.vr-mode {
   pointer-events: auto;
 }
 
+:local(.panel) {
+  position: absolute;
+  width:400px;
+  left: 50%;
+  margin-left: -200px;
+  bottom: 24px;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  padding: 5px;
+  background-color: $hud-panel-background;
+  border-radius: 300px;
+}
+:local(.iconEmoji) {
+  appearance: none;
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  outline-style: none;
```

Comment:
I think these four properties are unnecessary, since the element is already just a `div`